shell/platform/android/io/flutter/embedding/engine/android/FlutterView.java

```diff
@@ -174,8 +179,22 @@ protected void onSizeChanged(int width, int height, int oldWidth, int oldHeight)
    */
   @Override
   public final WindowInsets onApplyWindowInsets(WindowInsets insets) {
-    // TODO(mattcarroll): hookup to Flutter metrics.
-    return insets;
+    WindowInsets newInsets = super.onApplyWindowInsets(insets);
+
+    // Status bar, left/right system insets partially obscure content (padding).
```

Comment:
I'd change this comment to something like:
  Status bar (top) and left/right system insets should partially obscure the content (padding)

Reply:
Done